xbmc/platform/darwin/DarwinUtils.mm

```diff
@@ -285,7 +285,7 @@ bool CFStringRefToStringWithEncoding(CFStringRef source, std::string &destinatio
             manufName = static_cast<const char*>([NSString stringWithString:(__bridge NSString*)manufacturer].UTF8String);
           else if (typeId == CFDataGetTypeID())
           {
-            manufName.assign((const char*)CFDataGetBytePtr((CFDataRef)manufacturer), CFDataGetLength((CFDataRef)manufacturer));
+            manufName.assign(reinterpret_cast<const char*>(CFDataGetBytePtr((CFDataRef)manufacturer), CFDataGetLength((CFDataRef)manufacturer)));
```

Comment:
This is wrong, because originally only the return value of `CFDataGetBytePtr(...)` was affected by the cast to `const char*`.

Reply:
I reverted this because im a bit busy with work, but had a quick second to realise what id done. Have pushed a proper reinterpret_cast only encasing the CFDataGetBytePtr.
If you want to just check i did that right this time, sorry about that.